crates/typst/src/text/deco.rs

```diff
@@ -276,12 +276,12 @@ impl Show for Packed<StrikeElem> {
 #[elem(Show)]
 pub struct HighlightElem {
     /// The color to highlight the text with.
-    /// (Default: 0xffff5f)
+    /// (Default: #fffd11a1)
```

Comment:
We can just get rid of this line entirely. It's already displayed more nicely below.